server/middlewares/frontendMiddleware.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const path = require('path');
+
+// Dev middleware
+const addDevMiddlewares = (app, options) => {
+  const webpackDevMiddleware = require('webpack-dev-middleware');
+  const webpackHotMiddleware = require('webpack-hot-middleware');
```

Comment:
Ahh, didn't even know `--production` exists, that's hella annoying, might actually bite us in other places? Good point of thinking about that though!